packages/bot-web-ui/src/components/dashboard/bot-builder/toolbar/toolbar-icon.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { popover_zindex } from 'Constants/z-indexes';
+import { Icon, Popover } from '@deriv/components';
+
+type TToolbarIcon = {
+    popover_message: string;
+    icon: string;
+    icon_id: string;
+    iconOnClick: () => void;
```

Comment:
```suggestion
    icon_on_click: () => void;
```

Reply:
as far as I know if its a data variable we use snake case such as `icon_on_click` but if its a method we use camel case.
Kindly let me know if this isn't correct @michio-rework